### ISSUE TITLE
myNetConnectionObject.call

### ISSUE THREAD

**Issue by ciaoamigoschat** [aeinstein/webrtmp#1]
Hi great work thanks. Then tell me how can I also call a remoteMethod or receive a call remotely. In as3 it was the NetConnection.call().
Thank you

**Comment by aeinstein** [comment]
It should not be a problem, but I have only nginx rtmp module, this is not able to handle remote calls.
Have you a testserver (RED5 or similar) then I can test it.

**Comment by ciaoamigoschat** [comment]
I have Adobe Media Servers. But in production. I can create a server side application that sends and receives calls to you.
If you give me the ok today I'll prepare it for you.

**Comment by ciaoamigoschat** [comment]
News?

**Comment by aeinstein** [comment]
Yeah, I can try it. Install an Server App and give me the link. But it can take some days. I'm very busy

**Comment by ciaoamigoschat** [comment]
Thank you for your availability. Here is my AS3 code that connects to my server. 

var sephiroth:Object = new Object();
var amigoschat:NetConnection;
amigoschat=new NetConnection();
amigoschat.connect("rtmp://51.75.147.86/test/"+'room1','Pippo','2');
amigoschat.client =sephiroth;
amigoschat.addEventListener(NetStatusEvent.NET_STATUS, netStatuschat);
function netStatuschat(error:NetStatusEvent) 
{
        trace(error.info.code)
        if (error.info.code == 'NetConnection.Connect.Success')
        {
		        amigoschat.call('message', null, "Hi test");
        }
}
	
sephiroth.receive_message = function(msg:Object) {
       trace(msg)
}

Once the connection is established, it calls the service amigoschat.call('message', null, 'Hi test');. The service responds by calling sephiroth.receive_message. Naturally, you can also call the service from a button or wherever you prefer. amigoschat.call('message', null, 'Hi test');."

**Comment by aeinstein** [comment]
cool, I have just finished my last project. Next week I`ll see what I can do for you. I will write you when the test scenario is running

**Comment by ciaoamigoschat** [comment]
Thank you again so much.

**Comment by aeinstein** [comment]
Hi, I am ready to start. After reading your requirements, I will start with implementation of the NetConnection.call(). I will also implement SharedObjects, I have used them a lot in the past.
BTW: Parsing of a Data Message is already implemented.
Do you use AMF0 or AMF3 ??? 
Hope AMF0 because AMF3 is not implemented reflecting to the complexity of the login process.

Ok I just see that you can already make a call from JS.

I will test this with your Server
can you installl and config websockify as described in the readme ??

**Comment by ciaoamigoschat** [comment]
Hi, thanks so much. AMF0 is also fine.
I can't install websockify on that server as it's a very old Centos. I should get a new server. Can't launch websockify locally? Pointing to my server?

**Comment by aeinstein** [comment]
I will install it on my server pointing to yours. pls accept incoming from 95.216.12.85

**Comment by ciaoamigoschat** [comment]
Hi, I tried and it accepts every connection. Both AMF0 and AMF3.
...
amigoschat.objectEncoding = flash.net.ObjectEncoding.AMF0;
amigoschat.connect("rtmp://51.75.147.86/test/"+'room1','Pippo','2');
....

**Comment by ciaoamigoschat** [comment]
Hi, how's it going? You managed to connect to my server

**Comment by aeinstein** [comment]
sorry dude, I forget I am now on vacation. I have started the tests, now I am in France

**Comment by ciaoamigoschat** [comment]
Thanks for your availability of help. :)
